elements/pfe-text/src/pfe-text.scss

```diff
@@ -43,50 +47,69 @@
 ::slotted(h4),
 ::slotted(h5),
 ::slotted(h6),
-::slotted([pfe-text="headline-1"]),
-::slotted([pfe-text="headline-2"]),
-::slotted([pfe-text="headline-3"]),
-::slotted([pfe-text="headline-4"]),
-::slotted([pfe-text="headline-5"]),
-::slotted([pfe-text="headline-6"]) {
+::slotted([pfe-heading="xxl"]),
+::slotted([pfe-heading="xl"]),
+::slotted([pfe-heading="alpha"]),
+::slotted([pfe-heading="beta"]),
+::slotted([pfe-heading="gamma"]),
+::slotted([pfe-heading="delta"]),
+::slotted([pfe-heading="epsilon"]),
+::slotted([pfe-heading="zeta"]) {
     font-family: #{pfe-var(font-family--heading)};
-    line-height: #{pfe-var(line-height--heading)};
     margin-bottom: calc(#{pfe-var(content-spacer)}  * 1.1);
 }
 
 /* size */
+::slotted([pfe-heading="xxl"]) {
+    font-size: #{pfe-var(font-size--heading--xxl)};
+    font-weight: #{pfe-var(font-weight--light)};
+    line-height: #{pfe-var(line-height--heading)};
```

Comment:
The line-height from the design system is a bit more nuanced - the alpha has 1.3 but every other heading is 1.2.